pkg/loop/internal/ccip/test/onramp_test.go

```diff
@@ -65,12 +58,12 @@ func TestOnRampGRPC(t *testing.T) {
 	wg.Add(1)
 	go func() {
 		defer wg.Done()
-		<-shutdown
+		tests.RequireSignal(t, shutdown, "Failed to receive shutdown signal")
 		t.Log("shutting down server")
 		testServer.Stop()
 	}()
 	// create a price registry client
-	conn, err := grpc.Dial(addr, grpc.WithTransportCredentials(insecure.NewCredentials()))
+	conn, err := tests.SetupClientConnWithOptsAndTimeout(t, addr, grpc.WithTransportCredentials(insecure.NewCredentials()))
```

Comment:
maybe if we change this to ~ test.InsecureDial(t, addr) and mv the require.NoError & t.cleanup calls